src/pages/components/tabs/accessibility.mdx

```diff
@@ -18,79 +18,131 @@ import {
-have been identified and the list of open accessibility violations is available
-in the Carbon Component GitHub repository.
+![A user reaches the tablist with the Tab key. Pressing the Tab key again moves focus to a link inside the tabpanel's content](images/tab-accessibility-1a.png)
 
-### Automated test
+  </DoDont>
+  <DoDont type="do" caption="Where a tabpanel has no interactive items, the focus moves from the tablist to the entire tabpanel.">
+
+![A user tabs through a tablist. The focus goes from the tablist to the whole tabpanel, which contains only text.](images/tab-accessibility-1b.png)
+
+  </DoDont>
+</DoDontRow>
+
+Arrow keys are used to navigate between individual tab items in the tablist.
+When the end of the tablist is reached, the focus wraps to the opposite end of
+the list. For scrollable tablists, where the number of tabs exceeds the
+horizontal space, the keyboard navigation does not change. The user presses the
+`Left` or `Right` arrow keys, which moves the focus to the next tab item and
+where necessary scrolls the tablist to keep the selected item visible. For mouse
+users, clickable arrows appear at the end of the tablist to provide the same
+scrolling, but these are not needed for keyboard users and they are not in the
+focus order.
+
+<Row>
+
+<Column colLg={8}>
+
+![An image pair showing two views of the same tablist of 6 items. The user navigates between tab with left and right arrow keys. If the user presses the left arrow on the first item, the focus wraps around to item 6. Likewise arrowing right from tab item 6 moves to the first tab](images/tab-accessibility-4.png)
+
+<Caption>
+  Arrows keys move between tabs in the tablist (including wrapping from last
+  back to first). The tablist display scrolls automatically to keep the focused
+  tab visible.
+</Caption>
+
+</Column>
+</Row>
+
+Automatic and manual tablists differ in how the tab items are activated. The
+following illustration shows what will happen for each variant when a right
+arrow key is pressed with the Overview tab selected and focused.
+
+For automatic tablists, focus and selection are synchronized. When the user
+arrows to a tab, it is selected and the tabpanel under the tab is updated in
+real time.
+
+Manual tablists allow the user to arrow between the tab items without updating
+the tabpanel underneath. When the user right arrows, the Overview tab remains
+selected while focus moves to the Details tab. In order to select the Details
+tab (and update the tabpanel under the tab) the user would press `Enter` or
+`Space`.
+
+<Row>
+<Column colLg={8}>
+
+![Automatic and manual versions of a tablist with tabs called Overview, Details, and Support. In the first, the Details tab is selected and focused. In the second the Overview tab is still selected and the Details tab has a focus indicator](images/tab-accessibility-2.png)
+
+<Caption>
+  After pressing the `Right Arrow` key, the second tab is selected in an
```

Comment:
@laurenmrice What we could do is make an image pair, and split the caption between the two. You may recall that I suggested putting content in the tabpanel for these. That would give us an ability to have a bit more content in each image (which makes sense, I think)

Reply:
I split them out into two different images in the [image box folder ](https://ibm.ent.box.com/folder/222928628477)^

Reply:
Further to my comment on this pair including a tabpanel, I think the important thing is to including the shading that makes it clear which tabpanel is revealed. here's a snippet of what you have in the first image pair 
![image](https://github.com/carbon-design-system/carbon-website/assets/14298245/ce634149-8632-404f-8303-a7319d0876a7)
If you imagine combining that with the current image, so that we'd see that the first tab is called Overview and the second Details and then the focus is on  Details while the Overview tab is still selected, I think that would be helpful. I definitely think it would work better as a set of contained tabs, as opposed to Line
![image](https://github.com/carbon-design-system/carbon-website/assets/14298245/53c1ab6b-24d3-41bb-9976-b359c8d0e964)